{"uuid": "982dda80-31c4-4c4f-be53-836a7ec6c3c9", "children": ["65a8a00c-d2e0-4cb9-9a93-81e92df48625"], "befores": [{"name": "feed_explorer_agent", "status": "broken", "statusDetails": {"message": "KeyError: 'llama-3.2'\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/OA/OA/tests/agent_trajectory/feed_explore.py\", line 13, in feed_explorer_agent\n    llm = get_available_providers()[model]\n          ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^\n"}, "start": 1730426958706, "stop": 1730426960917}], "start": 1730426958706, "stop": 1730426964526}